clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -143,9 +143,23 @@ public boolean rename(Path path, Path path1) throws IOException {
     }
 
     @Override
-    public boolean delete(Path path, boolean b) throws IOException {
-        LOG.debug("$$$$$$$$$$$$$$$$$$$$$$$$$$$$ delete $$$$$$$$$$$$$$$$$$$$$$$$$$$$ ");
-        return false;
+    public boolean delete(Path path, boolean recursive) throws IOException {
+        LOG.debug("$$$$$$$$$$$$$$$$$$$$$$$$$$$$ Delete path {} - recursive {} $$$$$$$$$$$$$$$$$$$$$$$$$$$$",
+                path, recursive);
+
+        ObjectLocation loc = pathToObjectLocation(path);
```

Comment:
```suggestion
        ObjectLocation objectLoc = pathToObjectLocation(path);
```
(now that we have `stagingLoc`...)